Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md

```diff
@@ -0,0 +1,38 @@
+
+# Audio/Video in Google Chrome
+
+
+
+ _**Applies to:** Skype for Business 2015_
+
+At the moment we only support one incoming video channel in Google Chrome. 
```

Comment:
Maybe rephrase this, since this makes it sound like our fault and not a technological limitation imposed on us by WebRTC. It also makes it sound like at some point we're going to add this 'support,' which as far as I know isn't going to happen